### PR TITLE
Feature/dom updates

### LINKED ISSUE #56: DOM Manipulation
- [x]  Create domUpdates.js file
- [x] Transfer all functions that manipulate the DOM to file

### LINKED ISSUE #57: Modify API endpoints

### PR DESCRIPTION
**Pull request by mr-ryan12**
What (if any) features are you implementing?
- Move DOM manipulation files into domUpdates.js file

What (if anything) did you refactor?  
- Refactored functions for DRYness in both the scripts.js and domUpdates.js file

Were there any issues that arose?
- None at the time

Is there anything that you need from your teammate?
- Code review and merge pull request

Any other comments, questions, or concerns?
- Not at the moment

### CODE REVIEW

File: src/domUpdates.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+      page.innerHTML += `
+        <section class="individual-recipe-card">
+          <section class="recipe-card" id="${recipe.id}">
+            <img src="${recipe.image}" alt="${recipe.name}" class="recipe-card-image">
+            <i class="fa fa-heart heart-btn"></i>
+            <i class="fa fa-bookmark save-recipe-btn"></i>
+          </section>
+          <h2 class="recipe-card-title">${recipe.name}</h2>
+        </section>`
+    });
```
Comment (philalewis) [inline src/domUpdates.js:30]: Refactored to include another function as well in order to DRY up code.

File: src/domUpdates.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  },
+
+  updateToggle(recipeId, buttons, color) {
+    buttons.forEach(button => {
+      button.parentNode.id === recipeId ?
+        button.classList.toggle(`${color}`) : null;
+    })
+  },
+
+  updateButtons(list, user, buttons, color, category) {
```
Comment (oacortez) [inline src/domUpdates.js:87]: Thank you for making this function dynamic for updating color buttons

File: src/scripts.js [hunk 3]
```diff
@@ -236,55 +179,31 @@ function toggleCookbook(event) {
...
-
-function updateNavBarButtonColor() {
-  cookbookButton.classList.remove('beige');
-  allRecipesButton.classList.remove('beige');
-  favoritesButton.classList.remove('beige');
-  recipeRepo.filterState === 'all' ?
-    allRecipesButton.classList.add('beige') :
-    recipeRepo.filterState === 'favorites' ?
-      favoritesButton.classList.add('beige') : null;
-}
```
Comment (philalewis) [inline src/scripts.js:290]: Moved lots of code into the domUpdates file

**Review by philalewis — APPROVED** [review]

File: src/domUpdates.js [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+    allRecipesButton.classList.remove('grey');
+    favoritesButton.classList.remove('grey');
+    recipeRepo.filterState === 'all' ?
+      allRecipesButton.classList.add('grey') :
+      recipeRepo.filterState === 'favorites' ?
+        favoritesButton.classList.add('grey') : null;
+  }
+}
+
+export default domUpdates;
```
Comment (oacortez) [inline src/domUpdates.js:136]: Thanks for creating this domUpdates  file so our dom manipulation stays here and gets imported to our scripts file for more dry code

**Review by oacortez — APPROVED** [review]
Everything looks good! Will be merging this. Thanks!

**Pull request merged**